addons/mail/static/src/models/thread/thread.js

```diff
@@ -1968,8 +1969,12 @@ function factory(dependencies) {
         mass_mailing: attr({
             default: false,
         }),
-        members: many2many('mail.partner', {
-            inverse: 'memberThreads',
+        /**
+         * List of the members of the thread, contains thread specific information for each member
```

Comment:
"List of the" is overkill in doc as it is a `one2many`  (also not technically a List but a Set).
Also missing dot at the end.

Add in the doc that it only makes sense on channels.